### ISSUE TITLE
Release helm chart for nebula studio

### ISSUE THREAD

**Issue by jjsimps** [vesoft-inc/nebula-studio#361]
**Is your feature request related to a problem? Please describe.**
I have to clone the repo in order to install the chart. This makes automation (e.g. terraform) difficult.

**Describe the solution you'd like**
Have nebula studio available as a helm chart in a repository, instead of requiring a clone. For example, nebula-operator helm chart is available at `https://vesoft-inc.github.io/nebula-operator/charts`

**Describe alternatives you've considered**
Cloning as described in the docs.

**Comment by wey-gu** [comment]
Agreed, we may use GitHub page of this repo (https://vesoft-inc.github.io/nebula-studio/charts ).

cc @hetao92
